### PR TITLE
Adding validation for confirm button in wizard

### PR DESCRIPTION
**Pull request by ryelo**

### CODE REVIEW

**Comment by ryelo** [comment]
@jharting if you don't want to enforce a name, should I also remove the isRequired from the input field?

**Comment by jharting** [comment]
@ryelo yes please

**Comment by ryelo** [comment]
@karelhala this is still a WIP at the moment. So it's going to dismiss your review 😄

File: src/SmartComponents/Remediations/RemediationWizard.js [hunk 1]
```diff
@@ -207,7 +209,10 @@ class RemediationWizard extends Component {
     /*
      * State manipulation
      */
-    onNameChange = name => this.setState({ name });
+    onNameChange = name => this.setState({
+        name,
+        isValidated: (name.length ? true : false)
```
Comment (jharting) [inline src/SmartComponents/Remediations/RemediationWizard.js]: @ryelo as discussed name should not be required

Comment (ryelo) [reply src/SmartComponents/Remediations/RemediationWizard.js]: @jharting fixed!

**Comment by ryelo** [comment]
@karelhala @jharting can you review again?

**Pull request merged**